landscaper/gardenlet/pkg/gardenlet_chart_values.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkg
```

Comment:
A package named `pkg` doesn't seem ok. What the reader should expect from this pkg?
Ref https://blog.golang.org/package-names

Reply:
It contains all the source code besides the cobra command for the landscaper gardenlet binary. 
Isn't that the same structure as for all of the g/g code?

Reply:
Do you got a better suggestion?

Reply:
renamed it to ./landscaper/pkg/gardenlet and out the `reconcile` and `delete` functionality  under the `controller` package.

Reply:
But please raise those kind of concerns earlier next time as it always takes effort to re-test and validate.

Reply:
> But please raise those kind of concerns earlier next time as it always takes effort to re-test and validate.

I am not super human to spot each and every thing when the diff is `+25,135 −577`.

Reply:
yes I get it, but this PR (closed the original PR to make it more manageable) is open since Nov 9, 2020 last year already.